src/app/features/config/form-cfgs/sync-form.const.ts

```diff
@@ -47,11 +47,32 @@ export const SYNC_FORM: ConfigFormSection<SyncConfig> = {
             ? []
             : [{ label: SyncProvider.GoogleDrive, value: SyncProvider.GoogleDrive }]),
           { label: SyncProvider.WebDAV, value: SyncProvider.WebDAV },
-
-          ...(IS_ELECTRON
-            ? [{ label: SyncProvider.LocalFile, value: SyncProvider.LocalFile }]
-            : []),
+          { label: SyncProvider.LocalFile, value: SyncProvider.LocalFile },
```

Comment:
We still need to exclude the web platform, so we might do it like this:

```suggestion
      ...((IS_ELECTRON || IS_ANDROID_WEB_VIEW)
            ? [{ label: SyncProvider.LocalFile, value: SyncProvider.LocalFile }]
            : []),
```

Reply:
Thinking about it it, I wondered if we could also check here if the android version has the required interfaces available and only show the option then?

Reply:
Yes you are right about both points.

Reply:
Resolved with https://github.com/johannesjo/super-productivity/pull/2195/commits/03eab3c06f3aa3883d17ce04fcdbf24219d9e18d I guess.